pages/TodoPage.js

```diff
@@ -0,0 +1,21 @@
+export class TodoPage {
+    
+    constructor(page) {
+        this.page = page;
+        this.mainTitle = page.locator('[data-test-id="general-title"]')
+        this.taskTitle = page.locator('[data-test-id="title-field"]')
+        this.taskDescription = page.locator('[data-test-id="task-description"]')
+        this.addButton = page.locator('[data-test-id="add-button"]')
+    }
```

Comment:
You can use `page. getByTestId()` instead of locator and `data-test-id` inside.

Check this one: https://playwright.dev/docs/locators#locate-by-test-id